community/bubblewrap/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < realpath-workaround.patch
+
+./configure \
+    --prefix=/usr \
+    --mandir=/usr/share/man \
+    --infodir=/usr/share/info \
```

Comment:
This line can be removed. We don't support info pages out of the box.

Reply:
edited the line out.